ArcadiaParties.CQRS/Handlers/GetAssistantEmployeeHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using ArcadiaParties.CQRS.Queries;
+using ArcadiaParties.Data.Abstractions.DTOs;
+using ArcadiaParties.Data.Abstractions.Repositories;
+using MediatR;
+using System.IO;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArcadiaParties.CQRS.Handlers
+{
+    class GetAssistantEmployeeHandler : IRequestHandler<GetAssistantEmployeeQuery, AssistantEmployeeDTO>
+    {
+        private readonly IAssistantTokenRepository _token;
+        private readonly IHttpClientFactory _clientFactory;
+        private readonly IMediator _mediator;
+
+        public GetAssistantEmployeeHandler(IAssistantTokenRepository token, IMediator mediator, IHttpClientFactory clientFactory)
+        {
+            _token = token;
+            _mediator = mediator;
+            _clientFactory = clientFactory;
+        }
+
+        public async Task<AssistantEmployeeDTO> Handle(GetAssistantEmployeeQuery request, CancellationToken cancellationToken)
+        {
+            var query = new GetAssistantUserQuery();
+            var user = await _mediator.Send(query, cancellationToken);
+            var employeeId = user.EmployeeId;
```

Comment:
Should be passed in the mediatr request.

Reply:
Fixed